dev-python/dogpile-cache/dogpile-cache-0.9.1.ebuild

```diff
@@ -14,14 +14,11 @@ S="${WORKDIR}/dogpile.cache-${PV}"
 LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="amd64 ~arm arm64 x86"
-IUSE="test"
-RESTRICT="!test? ( test )"
 
 RDEPEND=""
-DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+DEPEND="
 	>=dev-python/decorator-4.0.0[${PYTHON_USEDEP}]
 	test? (
-		dev-python/pytest[${PYTHON_USEDEP}]
 		dev-python/pytest-cov[${PYTHON_USEDEP}]
```

Comment:
```suggestion
```

Did I ever tell you I hate pytest-cov? Try to sed it out if it's not really, really necessary. That is, if it's only used for coverage reporting.

Reply:
It wasn't python@ so I was trying not to dig too much, but let's forget about that if it's an objective improvement.

Not even a real dep => gone!